components/src/core/EmptyState/EmptyState.tsx

```diff
@@ -54,7 +58,11 @@ const EmptyStateRender: React.ForwardRefRenderFunction<unknown, EmptyStateProps>
                 {title}
             </Typography>
```

Comment:
What's the color of the title here? Eyeballing from the screenshot it looks like textSecondary to me. Should be textPrimary in both light and dark.

Reply:
The default typography is inherit, so it inherits from root, which is textPrimary (see line 26). The demo is using an outdated version of the theme. With the latest it looks like:

![image](https://user-images.githubusercontent.com/29152776/112018054-9a500e80-8b04-11eb-82bf-08932cdce01e.png)
![image](https://user-images.githubusercontent.com/29152776/112018114-aa67ee00-8b04-11eb-86b3-e09715511aa8.png)

Reply:
Then visually that screenshot looks awesome :)